x-pack/test/security_solution_cypress/cypress/e2e/investigations/alerts/alerts_cell_actions.cy.ts

```diff
@@ -60,10 +59,10 @@ describe('Alerts cell actions', { tags: ['@ess', '@serverless'] }, () => {
           .first()
           .invoke('text')
           .then((severityVal) => {
-            filterForAlertProperty(ALERT_TABLE_SEVERITY_VALUES, 0);
+            filterForAlertProperty(ALERT_TABLE_SEVERITY_VALUES, 1);
```

Comment:
why did the number change?

Reply:
because I changed the way the dom element was queried. With the simplification, it removed some extra unused dom elements which had a `data-test-subj` that was used. To keep things simple and not introduce new code, I used another existing `data-test-subj`, but that one is on multiple elements on the page (in the KPI chart first then in the alerts table).
It is not the most elegant way to solve this, but it was the only way I found without modifying application's code.